### PR TITLE
No icon for "Pipeline Argument Tab"

### PR DESCRIPTION
**Pull request by lak-proddev**
Migrated from #3437 #3439 #3440.

Signed-off-by: Lakshminarayana Nekkanti <narayana.nekkanti@gmail.com>

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java [hunk 1]
```diff
@@ -621,4 +628,16 @@ public void run() {
...
   }
+
+  @Override
+  public Image getImage() {
+    return image;
+  }
+
+  @Override
+  public void dispose() {
+    if (image != null)
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java]: google style requires braces even for single line ifs

Comment (lak-proddev) [reply plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java]: Updated

**Comment by lak-proddev** [comment]
Still, these are the failures or performance issues?
codecov/patch — 25% of diff hit (target 50%)

**Comment by elharo** [comment]
The codecov measurements don't block merging, though ideally we'd add some tests here. I'm afraid the Dataflow parts of the plugin are not as well covered as the rest of CT4E.

**Comment by lak-proddev** [comment]
Thanks. let me know any other issues that I can help.

**Pull request merged**